<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtQuadBezierArrowDialog</class>
 <widget class="QDialog" name="QtQuadBezierArrowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtQtuadBezierArrowDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_from">
     <property name="title">
      <string>From</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_from_x">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="box_from_x">
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_from_y">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="box_from_y">
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_mid">
     <property name="title">
      <string>Mid</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="box_mid_x">
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_mid_y">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_mid_x">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="box_mid_y">
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="box_has_tail">
        <property name="text">
         <string>Has tail arrow?</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="box_has_head">
        <property name="text">
         <string>Has head arrow?</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="box_normal_pen_width"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="box_focus_pen_width"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_normal_pen_width">
        <property name="text">
         <string>Normal pen width</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_focus_pen_width">
        <property name="text">
         <string>Focus pen width</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_to">
     <property name="title">
      <string>To</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_to_x">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="box_to_x">
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_to_y">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="box_to_y">
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
